docs/docs/addons-immutability-helpers.md

```diff
@@ -1,16 +1,27 @@
 ---
 id: update
 title: Immutability Helpers
-permalink: docs-old/update.html
+layout: docs
+category: Add-Ons
+permalink: docs/update.html
 prev: create-fragment.html
-next: pure-render-mixin.html
+next: perf.html
 ---
 
-React lets you use whatever style of data management you want, including mutation. However, if you can use immutable data in performance-critical parts of your application it's easy to implement a fast `shouldComponentUpdate()` method to significantly speed up your app.
+**Importing**
+
+```javascript
+import update from 'react-addons-update' // ES6
+var update = require('react-addons-update') // ES5
+```
+
```

Comment:
Let's add a note that this is legacy and we recommend https://github.com/kolodny/immutability-helper instead.